<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetAdminPro Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: transparent;
	padding: 0;
	margin: 0;
	color: #fff;
}

#centralwidget{
	background-image: url(:/images/images/BgClient.jpg);
}

#widget{
	background-color: rgb(9, 27, 68);
	border-radius:20px;
}

QLineEdit{
	background-color: rgb(9, 10, 37);
	padding: 5px 3px;
	border-radius:5px;
	
}
QPushButton{
	background-color:  rgb(9, 10, 37);
	padding: 10px 5px;
	border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>700</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="InformationPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="InfoLogo">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ClientResources.qrc">:/icons/icons/airplay.png</pixmap>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="InfoText2">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Connect Your PC To Server</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="InfoText">
             <property name="text">
              <string>Enter Your Information Below</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="UserInputInfo">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLineEdit" name="PcName">
                <property name="placeholderText">
                 <string>Enter PC Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="InfoClientIpAddr">
                <property name="placeholderText">
                 <string>Enter I.P. Address Of Client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="InfoServerIpAddr">
                <property name="placeholderText">
                 <string>Enter The I.P. Address Of Server</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="InfoServerPort">
                <property name="placeholderText">
                 <string>Enter The Port Number Of Server</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="InfoCheckBox">
             <property name="text">
              <string>Are The Above Information Correct</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="InfoConnectBTn">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="InfoConnectIssueBtn">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Click Here If Facing issue To Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="InfoCC">
             <property name="text">
              <string>CC Anjani Jha</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ClientResources.qrc"/>
 </resources>
 <connections/>
</ui>
